org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -574,44 +570,48 @@ public void reset() throws ScanningException {
 
 	/**
 	 * Blocks until not paused
+	 *
 	 * @return true if state has not been set to a rest one, i.e. we are still scanning.
 	 * @throws Exception
 	 */
 	@SuppressWarnings("squid:S2274")
 	private boolean checkPaused() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)